Basic database schema

vr2262 opened the pull request:
- A `Video` has a `url_id` and `video_metadata`.
- A `TrackMatch` has a `track_id`, a `match_id`, and whether the track is `audio` or `video` (the matching track is the opposite). It also has their scores and seek times.

vr2262 committed:
Define schema

[Closes #3]

vr2262 committed:
Add default empty password

myf commented on requirements.txt:
```diff
@@ -1,4 +1,6 @@
+psycopg2==2.6.2
+sqlalchemy==1.0.14
 tornado==4.4.1
 youtube_dl==2016.8.17
```
this (youtube-dl) will change at a very frequent rate, i think we should not pin it at 8.17

vr2262 replied:
Actually... you're right. In this case I think it makes sense to allow it to be whatever.

vr2262 committed:
Add JSONB metadata field

@myf

vr2262 committed:
Remove version peg for youtube-dl

https://github.com/MilliVolt/matcher/pull/5/files/4d4a4b43e8a114724ef7f431d21af8d2953f4963#r75603280
@myf

vr2262 commented:
brb

vr2262 committed:
Reshuffle schema

vr2262 committed:
Add settings.py.example

vr2262 commented:
@myf I changed `Track` to `Video` so that you don't have to make 2 entries into the table for every video.

vr2262 commented:
@myf this one?
